keras_retinanet/bin/build_hdf5.py

```diff
@@ -0,0 +1,115 @@
+import argparse
+
+import h5py
+import numpy as np
+from tqdm import tqdm
```

Comment:
https://github.com/fizyr/keras-retinanet/blob/da16864ad1b16175bedba28bce018e5b5c26462b/keras_retinanet/utils/eval.py#L26

https://github.com/fizyr/keras-retinanet/blob/da16864ad1b16175bedba28bce018e5b5c26462b/keras_retinanet/utils/coco_eval.py#L23

There are other scripts using `progressbar2` instead of `tqdm`. I think we should choose one of them.

Reply:
Agreed.
Already changed and pushed changes :)